spec/features/work_packages/share_spec.rb

```diff
@@ -44,11 +44,13 @@
   shared_let(:non_shared_project_user) { create(:user, firstname: 'Non Shared Project', lastname: 'User') }
   shared_let(:shared_project_user) { create(:user, firstname: 'Shared Project', lastname: 'User') }
   shared_let(:not_shared_yet_with_user) { create(:user, firstname: 'Not shared Yet', lastname: 'User') }
+  shared_let(:another_not_shared_yet_with_user) { create(:user, firstname: 'Not shared Yet', lastname: 'User') }
```

Comment:
Half of the flickering specs I found while local testing come from this:

The two not shared with users have the same name so when searching the autocompleter sometimes it picks `not_shared_yet_with_user` and sometimes `another_not_shared_yet_with_user`
```ruby
  shared_let(:not_shared_yet_with_user) { create(:user, firstname: 'Not shared Yet', lastname: 'User') }
  shared_let(:another_not_shared_yet_with_user) { create(:user, firstname: 'Not shared Yet', lastname: 'User') }
```

We could address this by modifying this user's name to
```suggestion
  shared_let(:another_not_shared_yet_with_user) { create(:user, firstname: 'Not shared Yet 2', lastname: 'User') }
```

This would also make it quite simple to understand what their natural position should be alphabetically if we ever want to set an expectation on it

Reply:
Ah, good catch! 👍